blocks/editable/format-toolbar/link-input.js

```diff
@@ -0,0 +1,187 @@
+/**
```

Comment:
👍 I like this proposal, it's there because it was extracted from the `format-toolbar` at first.